### PR TITLE
Implementation of sleep

### PR DESCRIPTION
**Pull request by Eike-Flath**
I implemented sleep (issue #24) as described by the [POSIX standard](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/sleep.html). I did not implement GNU features like being able to specify time intervals in other units (e.g. `1m`, `2h`, ...).

### CODE REVIEW

File: src/sleep/sleep.c [hunk 1]
```diff
@@ -1,6 +1,48 @@
...
+    }
+    unsigned int new_n = n * 10 + (arg[i] - '0');
+    if (new_n < n) { // overflow
+      fprintf(stderr, "sleep: time interval '%s' is too big\n", arg);
+      exit(EXIT_FAILURE);
+    }
+    n = new_n;
+  }
+  return n;
+}
```
Comment (Sigmanificient) [inline src/sleep/sleep.c:29]: Use `strtol` (C99, section `7.20.1.3`)

**Review by Sigmanificient — CHANGES_REQUESTED** [review]

File: src/sleep/sleep.c (hunk 1 above)
Comment (Eike-Flath) [reply src/sleep/sleep.c]: Thanks for the review! I implemented the suggested change (see my latest commit)

**Review by Sigmanificient — APPROVED** [review]
lgtm

**Review by proh14 — COMMENTED** [review]
thx ;)

**Pull request merged**